src/App.tsx

```diff
@@ -82,6 +83,10 @@ const App: React.FC = () => {
                 path={SCREENS.EDIT_MONITORING}
                 element={<EditMonitoring />}
               />
+              <Route
+                path={SCREENS.PROFESSOR_HOME}
+                element={<ProfessorHome />}
+              />
```

Comment:
A home tem que ser uma tela só e o conteúdo dela que muda de acordo com o tipo de usuário. Se a gente for criar uma URI por tipo de home, um professor vai poder abrir a home de aluno e vice-versa, o que é um bug.

Reply:
Resolvido